package/manifest.json

```diff
@@ -41,8 +41,8 @@
         "groupchat"
       ],
       "meetingSurfaces": [
-        "Stage",
-        "SidePanel"
+        "stage",

```

Comment:
I changed these to camelCase because the Teams is complaining it and the build wouldn't succeed if I don't change it...

Reply:
I also think the meetingSurfaces is replaced with context according to the [manifest schema reference](https://docs.microsoft.com/en-us/microsoftteams/platform/resources/schema/manifest-schema). Please let me know if I should update it.

Reply:
Revert these change. since these are values we use.